careerfairsystems/arkad-app-2018
# #72 Fix api error screen design

## Change description

Fixes bad design choice of error view when api is not loading companies.

## How to verify

Change the api-url of loadCompanies and then start app so that there are no companies cached. The progam should say that there is a network error and the a reload button should appear with a white background. 

## Issues fixed

This fixes #72
